spot-ml/src/main/scala/org/apache/spot/SuspiciousConnects.scala

```diff
@@ -43,16 +44,22 @@ object SuspiciousConnects {
         val sqlContext = new SQLContext(sparkContext)
         implicit val outputDelimiter = config.outputDelimiter
 
+        val inputDataFrame = InputOutputDataHandler.getInputDataFrame(sqlContext, config.inputPath, logger)
+          .getOrElse(sqlContext.emptyDataFrame)
+        if(inputDataFrame.rdd.isEmpty()) System.exit(0)
```

Comment:
The error was in InputOutputDataHandler.getInputDataFrame. I'm movining to SuspiciousConnects.

`  def getInputDataFrame(sqlContext: SQLContext, inputPath: String, logger: Logger): 

Option[DataFrame] ={

    try {

      logger.info("Loading data from: " + inputPath)

      Some(sqlContext.read.parquet(inputPath))

    } catch {

      case _ : Throwable => {

        logger.error(s"Couldn't read data from location $inputPath, please verify it's a valid location and that " +
          s"contains parquet files with a given schema and try again.")

        None
      }

    }

  }`